fleet-terraform-generator/internal/module/generate.go

```diff
@@ -176,6 +190,11 @@ func Generate(path, policyTemplateName, dataStreamName, inputName string) (*Terr
 	}
 	sort.Strings(allDataStreams)
 
+	PackagePolicyName := manifest.Name + "-" + dataStreamName + "-${var.fleet_data_stream_namespace}${var.fleet_policy_name_suffix}"
+	if len(strings.TrimSpace(dataStreamName)) == 0 {
```

Comment:
Did you find that the `TrimSpace` was necessary? Or would `if dataStreamName == "" {` be sufficient?

Reply:
Looks like this is not necessary so I've simplified it with your suggestino